syncopy/tests/test_resampledata.py

```diff
@@ -228,7 +228,7 @@ def test_rs_exceptions(self):
             resampledata(self.adata, method='nothing-real', resamplefs=self.fs // 2)
 
     def test_rs_selections(self):
-
+        np.random.seed(42)
         sel_dicts = helpers.mk_selection_dicts(nTrials=20,
                                                nChannels=2,
```

Comment:
It sets the seed for the `global`(!) numpy random number generator (`rng`), so it definitely affects the function calls afterwards. However, because of that, it may actually even be overkill or not that great (other parts of the code may change the global state, further down in the functions or somewhere else).

 The best solution would be to init a local `rng`, set its seed, and pass that seeded `rng` into the test functions, so you know exactly which `rng` is used where. I can implement that.

Reply:
as discussed, this may be needed due to a real bug, but fixing that is not part of this PR, so I will just leave it like this for now.